packages/frontend/src/components/Explorer/ExploreSideBar/index.tsx

```diff
@@ -0,0 +1,168 @@
+import { useApp } from '../../../providers/AppProvider';
+import { useExplorer } from '../../../providers/ExplorerProvider';
+import { TrackSection } from '../../../providers/TrackingProvider';
+import { SectionName } from '../../../types/Events';
+import AboutFooter from '../../AboutFooter';
+import { ExploreMenuItem } from '../../ExploreMenuItem';
+import { ShowErrorsButton } from '../../ShowErrorsButton';
+import ExplorePanel from '../ExplorePanel/index';
+import {
+    FooterWrapper,
+    MenuWrapper,
+    SearchWrapper,
+    SideBarDescription,
+    SideBarTitleWrapper,
+    SwitchFilter,
+} from './ExploreSideBar.styles';
+
+const SideBarLoadingState = () => (
+    <Menu large style={{ flex: 1 }}>
+        {[0, 1, 2, 3, 4].map((idx) => (
+            <React.Fragment key={idx}>
+                <MenuItem className="bp3-skeleton" text="Hello" />
+                <MenuDivider />
+            </React.Fragment>
+        ))}
+    </Menu>
+);
+const BasePanel = () => {
+    const history = useHistory();
+    const { projectUuid } = useParams<{ projectUuid: string }>();
+    const { errorLogs } = useApp();
+    const [search, setSearch] = useState<string>('');
+    const [filterExplores, toggleFilterExplores] = useToggle(true);
+    const exploresResult = useExplores(filterExplores);
+
+    const filteredTables = useMemo(() => {
+        if (exploresResult.data) {
+            if (search !== '') {
+                return new Fuse(Object.values(exploresResult.data), {
+                    keys: ['name', 'description'],
+                })
+                    .search(search)
+                    .map((res) => res.item);
+            }
+            return Object.values(exploresResult.data);
+        }
+        return [];
+    }, [exploresResult.data, search]);
+
+    if (exploresResult.data) {
+        return (
+            <>
+                <div>
+                    <SideBarTitleWrapper>
+                        <H3>Tables</H3>
+                    </SideBarTitleWrapper>
+                    <SideBarDescription>
+                        <Text>
+                            Select a table to start exploring your metrics
+                        </Text>
+                        <SearchWrapper>
+                            <InputGroup
+                                leftIcon="search"
+                                rightElement={
+                                    <Button
+                                        minimal
+                                        icon="cross"
+                                        onClick={() => setSearch('')}
+                                    />
+                                }
+                                placeholder="Search tables"
+                                value={search}
+                                onChange={(e) => setSearch(e.target.value)}
+                            />
+                        </SearchWrapper>
+                        <SwitchFilter
```

Comment:
Should this be inverted?
<img width="398" alt="Screenshot 2022-01-24 at 13 45 22" src="https://user-images.githubusercontent.com/11660098/150793965-380f51be-ef9e-42e6-a243-c388aa695eab.png">
<img width="375" alt="Screenshot 2022-01-24 at 13 45 30" src="https://user-images.githubusercontent.com/11660098/150793991-e43550b6-9690-468f-9c55-072b2a5c6c16.png">